### PR TITLE
resolve scheduler bug

### PR DESCRIPTION
**Pull request by tchaton**
https://github.com/nicolas-chaulet/torch-points3d/issues/328

### CODE REVIEW

File: torch_points3d/metrics/model_checkpoint.py [hunk 1]
```diff
@@ -287,5 +290,11 @@ def save_best_models_under_current_metrics(
                     current_stat["best_{}".format(metric_name)] = metric_value
                     models_to_save["best_{}".format(metric_name)] = state_dict
 
+        kwargs["model_props"] = {
+            "_num_epochs": model._num_epochs,  # type: ignore
```
Comment (nicolas-chaulet) [inline torch_points3d/metrics/model_checkpoint.py:298]: Could you remove the _ from their name since you access those directy? Also the base model defines some unused members
```
 self._latest_metrics = None
        self._latest_stage = None
        self._latest_epoch = None
```
Could you please delete them while you're at it?
